alert-policies/mexn/MongoDBUp.yml

```diff
@@ -19,20 +19,20 @@ valueFunction: SINGLE_VALUE
 terms:
   - priority: CRITICAL
     # Operator used to compare against the threshold.
-    operator: ABOVE
+    operator: NOT EQUAL TO
```

Comment:
This must be either ABOVE, BELOW or EQUALS

Reply:
Thank you @brammerl, It worked.. 👍